packages/demo-saas/src/BaseHomePage/components/Links.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+
+import Box from '@mui/material/Box';
+import { Link } from 'react-router-dom';
+
+const linkStyle = {
+  padding: '4px 10px',
+  fontSize: '15px',
+  textDecoration: 'none',
+  background: '#1976d2',
+  color: 'white',
+  fontFamily: '"Roboto","Helvetica","Arial",sans-serif',
+  border: '1px solid #1976d2',
+  borderRadius: '4px',
+};
+
+const links = [
+  { route: '/', label: 'Home page' },
+  { route: '/entitlements', label: 'Entitlements' },
+  { route: '/step-up-high-max-age', label: 'Step up high max age' },
+  { route: '/step-up-small-max-age', label: 'Step up small max age' },
+  { route: '/step-up-no-max-age', label: 'Step up no nax age' },
+  { route: '/step-up-modals', label: 'Step up modals' },
+  { route: '/step-up-hoc', label: 'Step up HOC' },
+  { route: '/step-up-transfer', label: 'Step up transfer' },
+  { route: '/unknown-route', label: 'Fallback route' },
+  { route: '/test', label: 'Old test' },
```

Comment:
you can reuse the strings from paths in routes - https://github.com/frontegg/frontegg-react/pull/1118/files#diff-4d1c0f1c53695db58cc90854af15a62e896842473b052336143c2a9f04788299R51-R58